src/providers/Theme.vue

```diff
@@ -25,6 +25,10 @@ export default defineComponent({
 
   // Shared properties
   --font-base: 'proxima nova', sans-serif;
+  --font-secondary: -apple-system, BlinkMacSystemFont, Helvetica, Arial,
```

Comment:
should we stick to web safe?

Reply:
This is web safe when considering the fallbacks. It's the font used in the designs. It's the macOS system font if using macOS. But I will check with Danielle if we should change this font. Let's leave it as is for now, it's not a blocker for this PR.